doc/man3/SSL_CONF_cmd.pod

```diff
@@ -58,9 +58,15 @@ Use server and not client preference order when determining which cipher suite,
 signature algorithm or elliptic curve to use for an incoming connection.
 Equivalent to B<SSL_OP_CIPHER_SERVER_PREFERENCE>. Only used by servers.
 
+=item B<-client_renegotiation>
+
+Allows servers to accept client-initiated renegotiation. Equivalent to
+setting B<SSL_OP_ALLOW_CLIENT_RENEGOTIATION>.
+Only used by servers.
+
 =item B<-legacyrenegotiation>
 
-permits the use of unsafe legacy renegotiation. Equivalent to setting
+Permits the use of unsafe legacy renegotiation. Equivalent to setting
```

Comment:
The uppercase of the first word is missing for other options.You might correct them all if you are fixing this particular one. But not required of course.

Reply:
There were only two uppercase fixes needed.

Reply:
Still need to push changes?

Reply:
i waited for 'make test' to finish :)